share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
  |  FreeBSD           |      |                    |                    |
  |   13.2             |      |                    |                    |
  |                    |      |                    |                    |
-FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -current
+ *--FreeBSD           |      |                    |                    |
+ |   14.0-ALPHA4      |      |                    |                    |
```

Comment:
Would _14.0 prerelease_ be better? 

To cover all ALPHA, BETA and other pre-release versions of 14.0.

Reply:
[`e11a48b` (#812)](https://github.com/freebsd/freebsd-src/pull/812/commits/e11a48ba1e6b7305af96bcc6ef62a7a74bdd6d0d)